techmatters/flex-plugins
# Upgrade Node version and @twilio/flex-plugin-scripts

## Description
Node v16 has reached end-of-life last year.

Changes in this PR:
- Node version: 16.20.0 =>  18.20.3
-  @twilio/flex-plugin-scripts: 6.3.3 => 6.4.1

The real motivation for this PR came from trying to integrate Storybook v8 into the repo.
This lib required node v18, and soon other libs and tools will do so.

### Verification steps
Verify that:
- Aselo still works fine ✅ 
- Automated Tests work ✅ 
- Create Tag action works ✅ 
- Deploy Action works  ✅ 

### AFTER YOU MERGE

1. Cut a release tag using the Github workflow. Wait for it to complete and notify in the #aselo-deploys Slack channel.
2. Comment on the ticket with the release tag version AND any additional instructions required to configure an environment to test the changes.
3. Only then move the ticket into the QA column in JIRA

You are responsible for ensuring the above steps are completed. If you move a ticket into QA without advising what version to test, the QA team will assume the latest tag has the changes. If it does not, the following confusion is on you! :-P
